sig-and-wg/sig-core/proposals/authorization-proxy.md

```diff
@@ -0,0 +1,29 @@
+# Authorization proxy
+
+
+## Abstract
+
+It is not possible to install kyma project on managed clusters like Google Kubernetes Engine (GKE) or Azure Kubernetes Service (AKS). To enable kyma-project adoption it has to run on any infrastructure supporting Kubernetes. Kyma requires access to kube-apiserver configuration and uses some kubernetes alpha features, that is **not allowed** in many managed clusters. Even if some providers allow for such configuration, it has to happen during cluster provisioning, what makes kyma installation process more complex.
+
+## Problems to solve
+
+1. The kube-apiserver component has to be configured to verify tokens coming from external OIDC provider (coming with kyma). Usually, it is done during installation by providing additional parameters: oidc-issuer-url, oidc-client-id, oidc-username-claim, etc. These parameters can't be set in GKE or AKS.
+2. We use some alpha features of kubernetes, that are not supported by managed clusters, e.g. PodPreset  from settings.k8s.io/v1alpha1
+
+
+## Proposal
+
+> Please read this document first to understand current authorization concept: https://github.com/kyma-project/kyma/blob/master/docs/authorization-and-authentication/docs/003-architecture.md
```

Comment:
~Please~ `Read [this](https://github.com/kyma-project/kyma/blob/master/docs/authorization-and-authentication/docs/003-architecture.md) document first to understand the current authorization concept.`